masonry/src/contexts.rs

```diff
@@ -851,11 +852,18 @@ impl LayoutCtx<'_> {
     ///
     /// This method will panic if [`WidgetPod::layout`] has not been called yet for
     /// the child.
+    #[track_caller]
     pub fn child_size(&self, child: &WidgetPod<impl Widget>) -> Size {
         self.assert_layout_done(child, "child_size");
         self.get_child_state(child).layout_rect().size()
     }
 
+    pub(crate) fn skip_layout(&mut self, child: &mut WidgetPod<impl Widget>) {
```

Comment:
Why is this `pub(crate)`? It seems possible that this could be usefully exposed externally.

I also think we should move this to between the private and public methods; I'd normally prefer the order to be public, pub(crate) and private, but that's not the extant style, which is fine.